src/webgpu/api/operation/rendering/depth_bias.spec.ts

```diff
@@ -261,3 +339,40 @@ g.test('depth_bias')
   .fn(async t => {
     t.runDepthBiasTest('depth32float', t.params);
   });
+
+g.test('depth_bias_24bit_format')
+  .desc(
+    `
+  Tests that a square with different depth bias values like 'positive', 'negative', 'infinity',
+  'slope', 'clamp', etc. is drawn as expected with 24 bit depth format.
```

Comment:
It's hard to write robust tests of depth test behavior without reading out the depth buffer values themselves. I think it would be better to test depth24plus by reading back the values (emulating the copy using a shader).

That said I think let's go ahead with adding these tests with a TODO here.

```suggestion
  'slope', 'clamp', etc. is drawn as expected with 24 bit depth format.

  TODO: Enhance these tests by reading back the depth (emulating the copy using texture sampling)
  and checking the result directly, like the non-24-bit depth tests, instead of just relying on
  whether the depth test passes or fails.
```